README.md

```diff
@@ -83,5 +83,41 @@ Credit: [@omundy](https://github.com/omundy)
 With `InventoryObject` selected, in the Canvas Scaler component, set `Reference Pixels Per Unit` to 32.
 
 
+#### Chapter 7, Page 263
+Credit: [@omundy](https://github.com/omundy)
+
+In the **The DamageCharacter() method** section, the code should be added to the `Enemy` class.
```

Comment:
Hey Owen - I think the text already specifies that this should be added to the `Enemy` class, unless I'm misunderstanding something here.

Reply:
The preceding section heading has the word Enemy Class in it, but then describes the Character class (and the player and enemy), so by the next section it wasn't not obvious to me. Many other code blocks that begin with "Add the following..." explicitly mention the class name, which I think is helpful.

Reply:
Ah, I see. Yes, I agree - clarification is a good idea here. Thanks for pointing it out.